tests/e2e/configurer/chain/queries.go

```diff
@@ -134,6 +135,24 @@ func (n *NodeConfig) QueryBalances(address string) (sdk.Coins, error) {
 	return balancesResp.GetBalances(), nil
 }
 
+func (n *NodeConfig) QueryEstimateSwapExactAmountIn(sender string, poolId uint64, denom, swapRouteDenoms, swapRoutePoolIds string, chainId string) (sdk.Int, error) {
```

Comment:
This query actually should not be here, but I implemented it at some point because I thought I needed it for e2e, I can leave it in this PR or create another one just so the work is not lost